### PR TITLE
Added github and gitlab urls to the default set

### PR DESCRIPTION
**Pull request by nielsbasjes**

### CODE REVIEW

File: patch-ext/background.js [hunk 1]
```diff
@@ -4,7 +4,11 @@ function load() {
 	if (!str) {
 		str = [
 			"https://issues[.]apache[.]org/jira/secure/attachment/[^/]*/.*[.]patch",
-			"https://issues[.]apache[.]org/jira/secure/attachment/[^/]*/.*[.]txt"
+			"https://issues[.]apache[.]org/jira/secure/attachment/[^/]*/.*[.]txt",
+			"https://github.com/[^/]*/[^/]*/commit/[^/]*[.]patch",
+			"https://github.com/[^/]*/[^/]*/commit/[^/]*[.]diff",
+			"https://gitlab.com/[^/]*/[^/]*/commit/[^/]*[.]patch",
+			"https://gitlab.com/[^/]*/[^/]*/commit/[^/]*[.]diff"
```
Comment (lifenstein) [inline patch-ext/background.js]: Wouldn't it be better to combine the four entries into just one with a group listing alternatives, like:
`https:\/\/git(hub|lab)[.]com\/.*[.](patch|diff)` ? 

I've added escaping for the forward slashes as well.

Comment (nielsbasjes) [inline patch-ext/background.js]: These are the default settings that you get when you initially load the plugin.
I personally prefer having regexes that are more readable for the end-user so they can more easily choose which to keep/change.

When do I see this extra url?

Why not go for  something as simple as this?

    .*\.(patch|diff)

Comment (lifenstein) [inline patch-ext/background.js]: About the URL: It's an under-documented feature, but https://patch-diff.githubusercontent.com/raw/daviddengcn/patch-ext/pull/9.diff and https://patch-diff.githubusercontent.com/raw/daviddengcn/patch-ext/pull/9.patch are examples.

**Comment by daviddengcn** [comment]
Thanks for the PR

**Pull request merged**